Nodejs/Product/Nodejs/Snippets/1033/Nodejs/tryf.snippet

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>tryf</Title>
+      <Description>Code Snippet for a Try, Catch, Finally Statement</Description>
+      <Shortcut>tryf</Shortcut>
+      <Author>Microsoft Corporation</Author>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Literal>
+          <ID>e</ID>
+          <Default>e</Default>
+        </Literal>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[try {
+$end$
+}catch ($e$){
+
+}finally {

```

Comment:
add a space before the catch and finally blocks, and also make this a SurroundsWith snippet too.